NAMESPACE

```diff
@@ -24,6 +25,7 @@ export(git2r_credentials)
 export(git_protocol)
 export(git_sitrep)
 export(git_vaccinate)
+export(github_token)
```

Comment:
Newly exported for visibility. Also, it's a frequent default value for `auth_token` and I don't like to use unexported functions in a public signature.